### PR TITLE
Add change password page

### PR DESCRIPTION
**Pull request by xxroltsxx**
Add a change password page for (Super) Admins.

### CODE REVIEW

**Comment by alphartdev** [comment]
Hi, I don't really have lot of time currently and therefore I take a bit of time to process PR. I hope you don't mind, and wanted to thank you cause, you already did some PR and it's really helping both the project and me! I'll keep you tuned once I've taken a look at your PR.

**Comment by xxroltsxx** [comment]
I fully understand :)

File: application/models/admin_model.php [hunk 1]
```diff
@@ -52,6 +52,21 @@ public function createAccount($user, $password){
...
+	
+	public function updateAccount($user, $password){
+		if(strlen($password) < 6){
+			$answer = new AJAXAnswer("Error: the password must be 6 characters or longer", false);
+			return $answer->getJSON();
+		}
+
+		$salt = substr(md5(uniqid(rand(), true)), 0, 16);
+		$hash = hash("sha512", $password . $salt);
+		$query = $this->database->prepare("UPDATE BAT_web SET password = '$hash', salt = '$salt' WHERE user = '$user'");
```
Comment (alphartdev) [inline application/models/admin_model.php]: Just going to use Prepared Statement (or whatever it is called in PHP), to avoid sql injections.

**Comment by alphartdev** [comment]
Thanks again for that nice PR!

**Comment by xxroltsxx** [comment]
No problem! Thank you for your amazing system!

**Pull request merged**